test/README.md

```diff
@@ -201,6 +201,28 @@ The following is a list of all available settings:
                          default: (none). Test files with this variable will be ignored unless
                          the D_OBJC environment variable is set to "1"
 
+    EXTRA_ARTIFACT:     artifact used in the test (configuration) which needs to be generated
+                        by invoking a specified command before running the test.
+
+                        Syntax: <artifact> => <command>
+
+                        Both <artifact> and <command> can use bash-like variables as defined below
+                        to handle platform specific build whereas <command> also can use $@ to
+                        refer to <artifact>. The ${DMD} variable also contain the appropriate model
+                        and PIC flags by default (as denoted by MODEL/PIC).
+
+                        Generated artifacts can be referenced in REQUIRED_ARGS by using $@[<i>],
+                        where <i> is the zero-based index among all artifacts in lexical order.
+
+                        Example:
+                        EXTRA_ARTIFACT: foo.lib => ${DMD} -lib -of=$@ foo.d
```

Comment:
Is there a reason for choosing this symbol?
I'm asking because logically the arrow should point to the left as that's the foo.lib doesn't generate DMD, but the other way around.

Reply:
Not really, it was just the first thing that came to mind (I read it as "For this, do that").

But I don't mind replacing it with a better suggestion.

Reply:
I don't really have a better suggestion (maybe someone else?). The only two ideas I have are `=` and `<=`

Reply:
Changed it to `=`